### PR TITLE
docs: added landing page for PDK docs

### PR DESCRIPTION
**Pull request by swap-aws**
Fixes #

### CODE REVIEW

File: public/docs/content/.pages.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 nav:
-  - Welcome: welcome
+  - Overview: overview
+  - Getting started: getting_started
```
Comment (agdimech) [inline public/docs/content/.pages.yml:4]: I think we should rename this to `Contributing` and move it below API Reference as this pertains to developers who want to contribute.

File: public/docs/content/.pages.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
   - Developer Guides: developer_guides
-  - Walkthroughs: walkthroughs
   - API Reference: api
-  - FAQ: faqs
+  - Release Notes: release_notes
+  - Troubleshooting: faqs
```
Comment (agdimech) [inline public/docs/content/.pages.yml]: Lets remove this and move all the FAQ's directly under the package they pertain to.

File: public/docs/content/getting_started/index.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Getting started
+
```
Comment (agdimech) [inline public/docs/content/getting_started/index.md:2]: This is empty and can be removed.

File: public/docs/content/getting_started/.pages.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+  nav:
+    - Getting started: index.md
```
Comment (agdimech) [inline public/docs/content/getting_started/.pages.yml:3]: This is empty and can be removed.

File: public/docs/content/getting_started/.pages.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    - Overview of core modules: core_modules.md
+    - Project structure: project_structure.md
+    - Creating a new construct: create_new_construct.md
+    - How to contribute: how_to_contribute.md
```
Comment (agdimech) [inline public/docs/content/getting_started/.pages.yml]: This should be moved to the top.

File: public/docs/content/overview/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+This combination of project management, build execution, and CDK constructs reduce the effort involved in prototyping new ideas on AWS, and allow developers to seamlessly grow these ideas beyond the prototype phase without requiring a complete re-implementation of the original solution. In addition to the foundation of project configuration and build dependency management provided by the nx-monorepo project, PDK also provides a number of other projen projects (some of which implement common infrastructure patterns using CDK) and stand-alone CDK constructs, which are described below. Note: PDK is currently pre-release, targeting version 1.0 for mid-2023.
+
+## About core modules
+
+The AWS Prototyping SDK (PDK) provides building blocks for common patterns to help you manage and build your projects. Before you get started with PDK, this is a introduction of the core modules.
+
+- nx-monorepo
```
Comment (agdimech) [inline public/docs/content/overview/index.md:13]: descriptions for each of these modules is missing.

File: public/docs/content/overview/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- static-website
+- cloudscape-react-ts-website
+- type-safe-api
+- pipeline
+- pdk-nag
+- cdk-graph
+- identity
+
+## Prerequisites
```
Comment (agdimech) [inline public/docs/content/overview/index.md]: This should be moved to a getting started section.

**Review by agdimech — COMMENTED** [review]
Great start - love the new landing page.

File: public/docs/content/.pages.yml [hunk 8]
```diff
@@ -1,7 +1,8 @@
 ---
 nav:
-  - Welcome: welcome
+  - Overview: overview
+  - Getting started: getting_started
   - Developer Guides: developer_guides
-  - Walkthroughs: walkthroughs
   - API Reference: api
-  - FAQ: faqs
+  - Release Notes: release_notes
```
Comment (agdimech) [inline public/docs/content/.pages.yml]: The content for this should link to: https://github.com/aws/aws-prototyping-sdk/releases

Comment (swap-aws) [reply public/docs/content/.pages.yml]: Updated

File: public/docs/content/.pages.yml (hunk 1 above)
Comment (swap-aws) [reply public/docs/content/.pages.yml:4]: recommendation: I'd separate the contributing from the `getting_started` if that's equally important. Absolutely ok to have it as a top level menu item, and another box on the landing page (the troubleshooting one will be gone anyways)

File: public/docs/content/getting_started/.pages.yml (hunk 5 above)
Comment (swap-aws) [reply public/docs/content/getting_started/.pages.yml]: See comment above about moving it to a different menu item

File: public/docs/content/overview/index.md (hunk 6 above)
Comment (swap-aws) [reply public/docs/content/overview/index.md:13]: Added a link to the `core_modules` page

File: public/docs/content/overview/index.md (hunk 7 above)
Comment (swap-aws) [reply public/docs/content/overview/index.md]: Updated

File: public/docs/content/.pages.yml (hunk 8 above)
Comment (swap-aws) [reply public/docs/content/.pages.yml]: resolved

File: public/docs/content/getting_started/.pages.yml (hunk 4 above)
Comment (swap-aws) [reply public/docs/content/getting_started/.pages.yml:3]: comment: I'd like to add summary content here

File: public/docs/content/getting_started/index.md (hunk 3 above)
Comment (swap-aws) [reply public/docs/content/getting_started/index.md:2]: comment: I'd like to add summary content here

**Comment by agdimech** [comment]
Closing as I have incorporated these changes into a seperate PR.

**Pull request closed without merging**